app/models/miq_server/update_management.rb

```diff
@@ -141,61 +110,8 @@ def enable_repos
     end
   end
 
-  def cfme_available_update
-    version_available = parse_product_version_number(MiqDatabase.first.cfme_version_available)
-    version_running   = parse_product_version_number(MiqServer.my_server.version)
-
-    version_available.keys.each do |version_component|
-      if version_available[version_component].to_i > version_running[version_component].to_i
-        return version_component.to_s
-      end
-    end
-
-    nil
-  end
-
-  def check_updates
-    _log.info("Checking for platform updates...")
-    check_platform_updates
-
-    _log.info("Checking for postgres updates...")
-    check_postgres_updates
-
-    _log.info("Checking for %{product} updates..." % {:product => Vmdb::Appliance.PRODUCT_NAME})
-    check_cfme_version_available
-
-    _log.info("Checking for updates... Complete")
-  end
-
-  def apply_updates
-    check_updates
-    return unless updates_available?
-
-    import_gpg_certificates
-
-    _log.info("Applying Updates, Services will restart when complete.")
-
-    # MiqDatabase.cfme_package_name will update only the ManageIQ package tree.  (Won't disturb the database)
-    # "" will update everything
-    packages_to_update = EvmDatabase.local? ? [MiqDatabase.cfme_package_name] : []
-    File.write(UPDATE_FILE, packages_to_update.join(" "))
-  end
-
   private
 
-  def check_platform_updates
-    update(:updates_available => LinuxAdmin::Yum.updates_available?, :last_update_check => Time.now.utc)
-  end
-
-  def check_postgres_updates
-    MiqDatabase.first.update(:postgres_update_available => LinuxAdmin::Yum.updates_available?(MiqDatabase.postgres_package_name))
-  end
-
-  def check_cfme_version_available
-    cfme = MiqDatabase.cfme_package_name
-    MiqDatabase.first.update(:cfme_version_available => LinuxAdmin::Yum.version_available(cfme)[cfme])
-  end
-
```

Comment:
I'm late to the game but I can't help but notice the next two methods seem like they're on the chopping block too.  I think they're both for handling updates in a similar way to this whole PR.

We can do lit later if there's some other detangling that needs to be done first.

Reply:
file is deleted. thanks